### PR TITLE
Updates and fixes for MOM6 in cime

### PR DESCRIPTION
**Pull request by alperaltuntas**
Several updates and fixes for MOM6 in cime xml files, drivers, and dice:

- add new resolution/domains/maps for JRA-forced MOM6 runs
- add new resolution/domains/maps for fully coupled MOM6 runs
- update existing MOM6 domains/maps to be compatible with the latest MOM6 2/3deg grid.
- update lat/lon consistency check such that  ((mesh coords) - (input domain file coordinates)) modulo 360 is checked to be less than epsilon.
- relax EPS_OGRID
- update CPL_ALBAV for MOM6
- fix memory leak in med_phases_prep_ocn_mod.F90

Test suite: aux_mom on izumi/intel,  ./scripts_regression_tests.py --fast
Test baseline: 
Test namelist changes: 
Test status: b4b, all pass 

User interface changes?: N

Update gh-pages html (Y/N)?: N

Code review:

### CODE REVIEW

**Comment by alperaltuntas** [comment]
Testing underway... Will update status to "Ready for review" once all tests pass.

**Comment by alperaltuntas** [comment]
The conflicts are due to CMEPS split. Not sure how cime and CMEPS get checked out after the split. (Is CMEPS now a new external of cesm or is it an external within cime?) I'll let @mvertens update this PR to resolve the conflicts.

File: src/drivers/nuopc/cime_config/config_component_cesm.xml [hunk 1]
```diff
@@ -324,6 +324,7 @@
     <default_value>$ATM_NCPL</default_value>
```
Comment (jedwards4b) [inline src/drivers/nuopc/cime_config/config_component_cesm.xml]: Remove the files under src/drivers/nuopc from this PR and create a separate PR to the CMEPS git.

**Comment by alperaltuntas** [comment]
> Remove the files under src/drivers/nuopc from this PR and create a separate PR to the CMEPS git.

I don't know how to properly check out cime and cmeps separately. Can you point me to the instructions? See my previous comment where I ask "Is CMEPS now a new external of cesm or is it an external within cime?"

**Comment by jedwards4b** [comment]
To test remove the files under src/drvers/nuopc and git clone https://github.com/ESCOMP/CMEPS.git
from directory src/drivers/nuopc.  Replace the files you are changing with your own and run tests.
Then submit a PR to CMEPS as well as here.  The CMEPS external is pulled into the UFSCOMP repo but not yet into cesm.   I think we can resolve that issue after your PR's are in.

**Comment by alperaltuntas** [comment]
I tried checking out latest cime and cmeps master branches as you described. Even without merging my changes, I cannot build SMS_Vnuopc.T62_t061.GMOM. So, it looks like cime master and cmeps master are incompatible. Which tags/commits should I check out?

testroot with failing build:  /scratch/cluster/altuntas/cesm.tests/SMS_Vnuopc.T62_t061.GMOM.izumi_intel.C.20190628_173534_dt278d

**Comment by jedwards4b** [comment]
This due to a recent change on the e3sm side.  You need to add a 0 at /home/altuntas/cesm.sandboxes/UFSCOMP.sbx/cime/src/drivers/nuopc/mediator/med_phases_aofluxes_mod.F90 line 508:
< aoflux%duu10n, ustar_sv=aoflux%ustar, re_sv=aoflux%re, ssq_sv=aoflux%ssq, &
> 0, aoflux%duu10n, ustar_sv=aoflux%ustar, re_sv=aoflux%re, ssq_sv=aoflux%ssq, &

**Comment by jedwards4b** [comment]
@alperaltuntas went on vacation prior to being able to resolve conflicts, we will close this and try again when he returns.

**Pull request merged**